lib/main.js

```diff
@@ -379,4 +389,11 @@ module.exports = class Client {
         const filename = this.bundleFilename(hash, type);
         return url.resolve(prefix, filename);
     }
+
+    async bundlingComplete(feedHashes, options = {}) {
+        const uri = await this.bundleURL(feedHashes, options);
+        if (!uri) return true;
```

Comment:
can this be cached so we avoid having to hit the server multiple times if we know the answer?

Reply:
Sounds like a good best practice type idea. In the interests of staying focused on the podium SDKs i might create an issue for it rather than do it now.